bpf/maps_definition.h

```diff
@@ -18,4 +18,17 @@ struct {
     __uint(map_flags, BPF_F_NO_PREALLOC);
 } aggregated_flows SEC(".maps");
 
+// Common hashmap to keep track of all flow sequences.
+// LRU hashmap is used because if some syn packet is received but ack is not
+// then the hashmap entry will need to be evicted
+// Key is flow_seq_id which is standard 4 tuple and a sequence id
+//     sequence id is specific to the type of transport protocol
+// Value is u64 which represents the occurrence timestamp of the packet.
+struct {
+    __uint(type, BPF_MAP_TYPE_LRU_HASH);
+    __uint(max_entries, 1 << 20);   // Will take around 64MB of space.
+    __type(key, flow_seq_id);
+    __type(value, u64);
+} flow_sequences SEC(".maps");
```

Comment:
pls collect cpu and memory profile when the feature is enabled and add it here for reference u can do that by bring up a cluster and check memory and cpu metrics before and after

Reply:
BTW the way LRU evicts is when the table reach its capacity
```
The BPF_MAP_TYPE_LRU_HASH and BPF_MAP_TYPE_LRU_PERCPU_HASH variants add LRU semantics to their respective hash tables. An LRU hash will automatically evict the least recently used entries when the hash table reaches capacity. An LRU hash maintains an internal LRU list that is used to select elements for eviction. This internal LRU list is shared across CPUs but it is possible to request a per CPU LRU list with the BPF_F_NO_COMMON_LRU flag when calling bpf_map_create.
```

Reply:
Did you mean something like a QE run?

Reply:
no this is a run u can do urself on cluster-bot

Reply:
@msherif1234 Please check the PR description. Added the numbers

Reply:
I will need clarifications on what u ran and associated the graphs I normally just label so what I expected main vs rtt off vs rtt on in the same graph for cpu and memory to make it easier to tell if there are any impacts if any

Reply:
@msherif1234 I noticed that the graphs were not tagged. Updated the PR description with new tagged graphs.

Reply:
Resolving this. Thanks.